### PR TITLE
Add GUAC contribution application

### PR DESCRIPTION
**Pull request by mlieberman85**
Application for GUAC to be adopted.

### CODE REVIEW

File: process/project-lifecycle-documents/guac_sandbox.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* "lumjjb", "Brandon Lum", "Google"
+* "pxp928", "Parth Patel", "Kusari"
+* "SantiagoTorres", "Santiago Torres", "Purdue University"
+* "rgreinho", "Rémy Greinhofer", "Citi"
+
+### Mission of the project
+  * GUAC (Graph for Understanding Artifact Composition) aims to fill in the gaps by ingesting software metadata, like SBOMs, SLSA attestations, and mapping out relationships between software. When you know how one piece of software affects another, you’ll be able to fully understand your software security position and act as needed. GUAC gives you directed, actionable insights into the security of your software supply chain.
+
+### Project adoption
+I will include quotes from adopters:
```
Comment (lehors) [inline process/project-lifecycle-documents/guac_sandbox.md:16]:
```suggestion
Here are quotes from adopters: 
```

File: process/project-lifecycle-documents/guac_sandbox.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  * GUAC came along as an open-source software at the right time helping us pivot away from building a bespoke solution and involving ourselves with the best minds behind the project. The value we see with GUAC is its flexibility and plugin architecture leading up to helping the users achieve compliance at different levels. The biggest benefit of GUAC has been producing it in the open with a widespread community behind it, from Google to Kusari and others. As the industry progresses, the threats to the software supply chain will become more complex, and relying on a tool backed by people with many years of experience in the area would make things easier for Guidewire to consume. -- Anoop Gopalakrishnan, Guidewire Software
+
+### Governance
+  * https://github.com/guacsec/guac/blob/main/GOVERNANCE.md
+
+### IP policy and licensing due dilligence
+  * TAC docs make this process unclear. We will work with LF on any due diligence, just point us in the right direction.
```
Comment (lehors) [inline process/project-lifecycle-documents/guac_sandbox.md:26]: This section needs to be updated.

**Review by lehors — CHANGES_REQUESTED** [review]
Looks good overall. Just need to update the IP section once it's completed.

**Review by lehors — APPROVED** [review]

**Comment by lumjjb** [comment]
Remove /hold

**Review by SecurityCRob — APPROVED** [review]

**Review by steiza — APPROVED** [review]

**Review by bobcallaway — APPROVED** [review]

**Review by torgo — APPROVED** [review]
lgtm (looks guac to me)

**Review by di — APPROVED** [review]

**Pull request merged**